test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -18,11 +18,14 @@ package benchmark
 
 // High Level Configuration for all predicates and priorities.
 type schedulerPerfConfig struct {
+	NodeCount    int // It means if the testcase has 1000 nodes, out of them only nodeCount will have needed predicates or priorities.
+	PodCount     int // It means if the testcase has 1000 pods, out of them only podCount will have needed predicates or priorities.
 	NodeAffinity *nodeAffinity
+	// TODO: Other predicates and priorities to be added here.
 }
 
 // nodeAffinity priority configuration details.
 type nodeAffinity struct {
-	numGroups       int    // number of Node-Pod sets with Pods NodeAffinity matching given Nodes.
 	nodeAffinityKey string // Node Selection Key.
+	LabelCount      int    // number of labels to be added to each node or pod.
```

Comment:
I am not able to find the documentation for that. I will check and update @jayunit100